specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2021-01-01-preview/examples/DataPolicy_List.json

```diff
@@ -0,0 +1,114 @@
+{
+  "parameters": {
+    "Endpoint": "{Endpoint}",
+    "api-version": "2021-01-01-preview"
```

Comment:
2022?

Reply:
Thanks, my team is revisiting this.

Reply:
We have updated the version in this PR to 2022-02-01-preview